test/serializable_resource_test.rb

```diff
@@ -71,6 +71,13 @@ def test_serializable_resource_with_collection_containing_errors
         }
         assert_equal serializable_resource.as_json(options), expected_response_document
       end
+
+      def test_serializable_resource_when_no_serializer_found
+        resource = ModelWithErrors.new
+        serializable_resource = ActiveModel::SerializableResource.new(resource)
+        # TODO define error to raise here
+        assert_raises { serializable_resource.as_json }
```

Comment:
maybe "No serializer found for #{model_name}"


Though, I wonder, do we have tests for no-serializer? cause it should just use the default json renderer shouldn't it?

or are we wanting to explicity force the use of serializers? I could see that being useful for JSON API.. but... idk.